README.md

```diff
@@ -24,7 +24,8 @@ const combinedComposables = composable0.compose(composable1, composable2, compos
 
 ### Descriptor
 
-**Composable descriptor** (or just **Descriptor**) is a meta data object. It's a collection of properties that contain the information necessary to create an object instance by a *composable*.
+**Composable descriptor** (or just **Descriptor**) is a meta data object. It's a collection of properties that contain
+the information necessary to create an object instance by a *composable*.
```

Comment:
Please use soft wrap instead of hard line feeds. Hard line feeds only look good *at the screen width that the author used*, and they look wrong *at every other screen width* (including diffs).